modules/scheduler/batch-job-template/variables.tf

```diff
@@ -166,15 +166,42 @@ variable "native_batch_mounting" {
   default     = true
 }
 
+# Deprecated, replaced by instance_image
+# tflint-ignore: terraform_unused_declarations
 variable "image" {
-  description = "Google Cloud Batch compute node image. Ignored if `instance_template` is provided."
-  type = object({
-    family  = string
-    project = string
-  })
+  description = "DEPRECATED: Google Cloud Batch compute node image. Ignored if `instance_template` is provided."
+  type        = any
+  default     = null
+
+  validation {
+    condition     = var.image == null
+    error_message = "The 'var.image' setting is deprecated, please use 'var.instance_image' with the fields 'project' and 'family' or 'name'."
+  }
+}
+
+variable "instance_image" {
+  description = <<-EOD
+    Google Cloud Batch compute node image. Ignored if `instance_template` is provided.
+
+    Expected Fields:
+    name: The name of the image. Mutually exclusive with family.
+    family: The image family to use. Mutually exclusive with name.
+    project: The project where the image is hosted.
+    EOD
+  type        = map(string)
   default = {
-    family  = "batch-hpc-centos-7-official"
-    project = "batch-custom-image"
+    project = "ubuntu-os-cloud"
+    family  = "ubuntu-2204-lts"
```

Comment:
We should keep the default image the batch hpc image.

Reply:
Thank you for catching that. I agree @cdunbar13 and it probably explains the failure.